acceptance-tests/scenarios.d/03.basic.failed.images.not.present.sh

```diff
@@ -51,12 +51,20 @@ function test_phase_run_images_not_present() {
         --platform linux/amd64 \
         --orchestrator docker-compose \
         --manifests-dir acceptance-tests/data/manifests-1-broken \
-        --application-name myapp0a || return 1
-    echo "images_not_present: checking install rc"
-    mender install "$artifact_file" && return 2 # we expect a failure
-    echo "images_not_present: checking for running containers"
-    docker ps -q | grep -q . && return 3 # and we expect nothing to be running
-    return 0
+        --application-name myapp0a # we expect a failure here now as we are using docker-compose to pull the images now
+
+    if [ $? -ne 0 ]; then
+        echo "images_not_present: app-gen artifact generation failed which is expected"
+        return 0
+    else
+        echo "Unexpected success, we were expecting failure"
+        return 1
+    fi
+    # echo "images_not_present: checking install rc"
+    # mender install "$artifact_file" && return 2 # we expect a failure
+    # echo "images_not_present: checking for running containers"
+    # docker ps -q | grep -q . && return 3 # and we expect nothing to be running
+    # return 0
```

Comment:
we probably should not modify the existing tests, why do we need to do it?

Reply:
in this test case, if you closely observe the existing flow, you will that we used image tag to pull image and then mender install it uses the docker compose to validate the image and expected a failure, as the image argument and docker compose totally defined with different image name. 

but right now, as per the requirement we are using the docker-compose also to pull an image, which means image argument is essentially becomes the optional one.

with that, during the artifact generation itself, the script will throw an error in the new logic.